types/react-big-calendar/index.d.ts

```diff
@@ -184,11 +184,18 @@ export interface ResourceHeaderProps {
     resource: object;
 }
 
+export interface DateCellWrapperProps {
+    range: Date[];
+    value: Date;
+    index: number;
```

Comment:
please double confirm `index` is really prop of `dateCellWrapper` I just had quick look in implementation and couldn't find it

https://github.com/jquense/react-big-calendar/blob/ad8defa643692911fd0b00c71b70de94715140a9/src/BackgroundCells.js#L53